### PR TITLE
docs: fix apiversion in InferenceService and ServingRuntime

### PR DESCRIPTION
**Pull request by dtrifiro**
Make apiVersion consistent across the docs example

### CODE REVIEW

**Comment by dtrifiro** [comment]
@bdattoma

**Comment by dtrifiro** [comment]
@Xaenalt could use your input here.

File: docs/caikit-servingruntime.yaml [hunk 1]
```diff
@@ -31,7 +31,7 @@ data:
                   connection:
                     hostname: localhost:8033
 ---
-apiVersion: serving.kserve.io/v1alpha1
+apiVersion: serving.kserve.io/v1beta1
```
Comment (bdattoma) [inline docs/caikit-servingruntime.yaml:34]: I think this should remain v1alpha1, while inference service should have v1beta1

Comment (bdattoma) [reply docs/caikit-servingruntime.yaml:34]: alternatively, I guess we could remove the line at all and it grabs the one installed in the cluster, doesn't it?

Comment (Xaenalt) [reply docs/caikit-servingruntime.yaml:34]: Yeah, let's match what KServe does today, or if we can omit it, that sounds like a good idea as well

Comment (dtrifiro) [reply docs/caikit-servingruntime.yaml]: Got rid of `apiVersion` declarations in the last push

Comment (bdattoma) [reply docs/caikit-servingruntime.yaml]: sorry Daniele, i just re-tested it and it must have an api version.

Comment (dtrifiro) [reply docs/caikit-servingruntime.yaml]: @bdattoma Updated again with the same format as the latest kserve docs. Let me know if it's fine.

**Review by bdattoma — APPROVED** [review]

**Pull request merged**